sdk/spring/azure-spring-boot/src/main/resources/META-INF/spring.factories

```diff
@@ -1,14 +1,14 @@
-org.springframework.boot.env.EnvironmentPostProcessor=com.microsoft.azure.spring.cloudfoundry.environment.VcapProcessor,\
-com.microsoft.azure.keyvault.spring.KeyVaultEnvironmentPostProcessor
-org.springframework.boot.autoconfigure.EnableAutoConfiguration=com.microsoft.azure.spring.autoconfigure.cosmosdb.CosmosAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.cosmosdb.CosmosDbRepositoriesAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.cosmosdb.CosmosDbReactiveRepositoriesAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.gremlin.GremlinAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.gremlin.GremlinRepositoriesAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.aad.AADAuthenticationFilterAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.aad.AADOAuth2AutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.b2c.AADB2CAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.metrics.AzureMonitorMetricsExportAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.jms.ServiceBusJMSAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.cosmosdb.CosmosHealthConfiguration,\
-com.microsoft.azure.keyvault.spring.KeyVaultHealthConfiguration
+org.springframework.boot.env.EnvironmentPostProcessor=com.azure.spring.cloudfoundry.environment.VcapProcessor,\
+com.azure.spring.KeyVaultEnvironmentPostProcessor
```

Comment:
the `KeyVaultEnvironmentPostProcessor` is under the `com.azure.keyvault.spring`, we have `com.azure.keyvault.spring` and `com.azure.spring` in azure-spring-boot`, don't confuse them